packages/jsii/tsconfig.json

```diff
@@ -48,10 +48,19 @@
     "inlineSources": true,                    /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
 
     /* Experimental Options */
-    "experimentalDecorators": true            /* Enables experimental support for ES7 decorators. */
+    "experimentalDecorators": true,           /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "composite": true                         /* Allow this project to be referenced by others. */
   },
+  "include": [
+    "**/*.ts"
+  ],
   "exclude": [
-    "test/**/neg.*.ts"
+    "**/*.d.ts",
+    "node_modules",
+    "test"
+  ],
+  "references": [
+    { "path": "../jsii-spec" }
```

Comment:
I don't like this one bit. Why are we referencing outside of the module tree?

Reply:
Well, this particular feature has been added in `typescript` *specifically* to improve the  developer experience around monorepos and composite projects... If you're not into the `../` reference, the best alternative is to reference via `./node_modules/jsii-spec` instead?

Reply:
`./node_modules` sounds much better for sure. But what happens if the reference directory doesn't have .ts files (i.e. it's not a mono-repo or we split those modules into separate repos)

Reply:
If there is no `tsconfig.json` in the pointed to directory, `typescript` will not like it. When the projects are "untangled", they need those project references edited out.